README.md

```diff
@@ -36,6 +36,7 @@ ExternalDNS' current release is `v0.5`. This version allows you to keep selected
 * [OpenStack Designate](https://docs.openstack.org/designate/latest/)
 * [PowerDNS](https://www.powerdns.com/)
 * [CoreDNS](https://coredns.io/)
+* [Oracle Cloud Infrastructure DNS](https://docs.cloud.oracle.com/iaas/Content/DNS/Concepts/dnszonemanagement.htm)
```

Comment:
Would be also nice if you add in section "Deploying to a Cluster" your tutorial

Reply:
Sorry this was maybe confusing, what I meant was in the README.md there's a section "Deploying to a Cluster". Could you add it there?

Reply:
Ah, right - no worries. Done now 🙂